src/main/resources/resources/logisim/strings/std/std_fr.properties

```diff
@@ -108,6 +108,7 @@ multiplierCarryOutTip = Retenue en sortie: le bit de poids le plus fort du produ
 multiplierComponent = Multiplicateur
 multiplierInputTip = Entrée: l'un des nombres à multiplier
 multiplierOutputTip = Sortie: le produit des entrées plus la retenue en entrée
+signedUnsignedOption = Signé/Non signé
```

Comment:
"Avec/sans signe"

Reply:
I think "signé/non signé" would be a better fit here; it's common to talk about "entier signé/non signé" or "valeurs signées/non signées". "Avec/sans signe" would, for me at least, be at least a little bit confusing on the exact purpose of the setting